src/transformers/tokenization_pegasus.py

```diff
@@ -14,11 +14,35 @@
 # limitations under the License.
 from typing import Dict, List, Optional
 
-from .file_utils import add_start_docstrings
-from .tokenization_reformer import ReformerTokenizer
+from .file_utils import add_start_docstrings, add_start_docstrings_to_callable
+from .tokenization_reformer import ReformerTokenizer, ReformerTokenizerFast
 from .tokenization_utils_base import PREPARE_SEQ2SEQ_BATCH_DOCSTRING, BatchEncoding
 
 
+SPIECE_UNDERLINE = "▁"
+
+####################################################
+# Mapping from the keyword arguments names of Tokenizer `__init__`
+# to file names for serializing Tokenizer instances
+####################################################
```

Comment:
Do we need these comments?

Reply:
They come from the template I believe. They can definitely be removed.